### DISCUSSION TITLE
z/OS Open Source Guild Meeting #11 - Monday, July 10 at 10am EST

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#351] [Guild]
### Agenda

Join us for the **11th z/OS Open Source Guild Meeting** where we are thrilled to have Vignesh S (@v1gnesh) as our guest speaker. Vignesh will be sharing many exciting updates with us!

The topics for the 11th Guild Meeting include:
* Highlights in the last month
* Access to _1 Password Teams_ for z/OS Open Tools contributors
* Demo of SSH auth and commit signing (on zOS) with 1 Password
* Demos of the latest tools:
  * powerline-go
  * direnv (if time permits)
* Keeping tools current
  * Leveraging _bump_ to keep tools current (if time permits)
* Demo of gotestsum (if time permits)
* Demos of c3270 and GNU Screen
* Feedback and Q/A session

Thank you for all the continued support!

### How to Join?
Join link: https://ibm.webex.com/ibm/j.php?MTID=m72857a0933fbcd1c58c116a13c9f1e7b 
Webinar number: 145 105 4264
Webinar password: QamxS8QX2m7 (72697879 from phones)

### Date and Time
* Monday, July 10 at 10am EST
* [ICS Calendar entry](https://github.com/ZOSOpenTools/meta/files/11960716/invite_guild11.ics.zip)

### Recording/Slide/Poll/Chat Log
* [Slides](https://github.com/ZOSOpenTools/meta/files/12003556/Open_Source_Guild_11_final.pptx)
* Related blog:
   * [Streamlining z/OS Interactions: Merging z/OS UNIX and TSO/ISPF with c3270
](https://igortodorovskiibm.github.io/blog/2023/06/25/c3270-on-zos/)
* [poll-results.pdf](https://github.com/ZOSOpenTools/meta/files/12004038/poll-results.pdf)
* [Recording](https://ibm.box.com/s/rdrfp4v1t1rsgzzfliwleytdfwyt0qn4)

**Comment by ijmitch** [comment]
[Vignesh's great video](https://youtu.be/C7M6FaTg_zQ)!

**Reply by IgorTodorovskiIBM** [reply]
Thank you!

**Reply by v1gnesh** [reply]
Thanks Ian, I've also marked the video public now.
